### ISSUE TITLE
[GPU related] Doesn't work on Fedora 35

### ISSUE THREAD

**Issue by anjannath** [flathub/com.brave.Browser#138]
### How to reproduce:
1. install com.brave.Browser using `flatpak install com.brave.Browser` (choose 1 or 2 for system or user mode installation)
2. try to start the browser from the gnome launcher, and screen starts to flicker and is unusable

### Following are the logs when using `flatpak run com.brave.Browser` in the terminal
```
➜  flatpak run com.brave.Browser 
Gtk-Message: 16:21:59.189: Failed to load module "canberra-gtk-module"
Gtk-Message: 16:21:59.189: Failed to load module "pk-gtk-module"
Gtk-Message: 16:21:59.189: Failed to load module "canberra-gtk-module"
Gtk-Message: 16:21:59.189: Failed to load module "pk-gtk-module"
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[54:54:0518/162159.304361:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.304526:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.304637:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.304791:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.304890:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305011:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305122:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305263:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305374:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305521:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305615:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305734:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305846:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.305958:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.306099:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.306231:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.306358:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.306481:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.306590:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.306727:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.306838:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307012:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307164:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307351:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307525:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307642:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307754:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307844:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.307935:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308070:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308169:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308255:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308342:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308453:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308566:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308663:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308760:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308846:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.308939:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
[54:54:0518/162159.309052:ERROR:gbm_wrapper.cc(275)] Failed to export buffer to dma_buf: No such file or directory (2)
```

let me know if more information/help is needed for debugging.

**Note** i am running fedora 35 on a Thinkpad P50 and i am using the nvidia non-free drivers

**Comment by anjannath** [comment]
Installing the rpm as instructed here: https://brave.com/linux/#fedora-28-centosrhel-8 works fine

**Comment by rany2** [comment]
Does this still happen in the latest Brave update? I presume this was fixed from some Zypak fix.

**Comment by anjannath** [comment]
@rany2 haven't tried the flatpak since, will give it a try today and let you knoe

**Comment by rany2** [comment]
@anjannath Any chance you could have a look at this today?

**Comment by kalrykh** [comment]
This may or may not be useful, but seeing as you're waiting on information, I thought I would let you know.  I'm on a different platform (aarch64) with a different GPU (Apple M1 Max), but in both Fedora 37 and Arch Linux ARM, the flatpak fails with a similar spew of information, whereas the rpm install (following instructions from https://brave.com/linux/) and AUR install for Arch both work fine and with GPU acceleration.  Not looking for assistance with my issue, as there's too much difference between my setup and the original issue reporter's and it would be best opening another issue if I were, but I thought knowing it was affecting more than just them might help.   Also, I had the same issue loading the GTK modules, which I resolved by changing the path for that GTK variable in FlatSeal to point to my natively installed GTK modules.  Both flatpak installs, on both Fedora and Arch, were done using the latest software in the past 24 hours.

Gtk-Message: 16:21:59.189: Failed to load module "canberra-gtk-module"
Gtk-Message: 16:21:59.189: Failed to load module "pk-gtk-module"
Gtk-Message: 16:21:59.189: Failed to load module "canberra-gtk-module"
Gtk-Message: 16:21:59.189: Failed to load module "pk-gtk-module"
